keyboards/ymdk/ymd40/v2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2021 James Young (@noroadsleft)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_ortho_4x12(
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
+        KC_ESC,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,
+        KC_CAPS, KC_LCTL, KC_LALT, KC_LGUI, KC_DEL,  KC_SPC,  KC_SPC,  RGB_MOD, KC_UP,   KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_ortho_4x12(
```

Comment:
There's no way to get to this layer.

Reply:
Ah, the joys of KBFirmware JSON distributions: can put keycodes in any layer you want, but no guarantee the layer can be accessed.

I think I'll keep this as `factory` and then modify it for a sensible `default`.